images/sd_charging_profile_clear.svg

```diff
@@ -1 +1 @@
h="10" x="871" y="183.125"/><path d="M12,69.5938 L12,94.5938 L928,94.5938 L928,79.5938 L918,69.5938 L12,69.5938 " fill="#FBFB77" filter="url(#f2r2eal91ixk9)" style="stroke: #A80036; stroke-width: 1.0;"/><path d="M918,69.5938 L918,79.5938 L928,79.5938 L918,69.5938 " fill="#FBFB77" style="stroke: #A80036; stroke-width: 1.0;"/><text fill="#000000" font-family="sans-serif" font-size="13" lengthAdjust="spacingAndGlyphs" textLength="178" x="374.75" y="86.6606">Ongoing Session with id=15</text><polygon fill="#A80036" points="681,120.8594,691,124.8594,681,128.8594,685,124.8594" style="stroke: #A80036; stroke-width: 1.0;"/><line style="stroke: #A80036; stroke-width: 1.0;" x1="71" x2="687" y1="124.8594" y2="124.8594"/><text fill="#000000" font-family="sans-serif" font-size="13" lengthAdjust="spacingAndGlyphs" textLength="603" x="78" y="119.7935">DELETE https://server.com/ocpi/2.2/chargingprofiles/15?response_url=https://client.com/12345</text><polygon fill="#A80036" points="77,149.9922,67,153.9922,77,157.9922,73,153.9922" style="stroke: #A80036; stroke-width: 1.0;"/><line style="stroke: #A80036; stroke-width: 1.0; stroke-dasharray: 2.0,2.0;" x1="71" x2="692" y1="153.9922" y2="153.9922"/><text fill="#000000" font-family="sans-serif" font-size="13" lengthAdjust="spacingAndGlyphs" textLength="499" x="83" y="148.9263">status_code = 1000, data: {ChargingProfileResponse { result = ACCEPTED }}</text><polygon fill="#A80036" points="859,179.125,869,183.125,859,187.125,863,183.125" style="stroke: #A80036; stroke-width: 1.0;"/><line style="stroke: #A80036; stroke-width: 1.0;" x1="698" x2="865" y1="183.125" y2="183.125"/><text fill="#000000" font-family="sans-serif" font-size="13" lengthAdjust="spacingAndGlyphs" textLength="143" x="705" y="178.0591">SetChargingProfile.req</text><polygon fill="#A80036" points="714,208.2578,704,212.2578,714,216.2578,710,212.2578" style="stroke: #A80036; stroke-width: 1.0;"/><line style="stroke: #A80036; stroke-width: 1.0;" x1="708" x2="875" y1="212.2578" y2="212.2578"/><text fill="#000000" font-family="sans-serif" font-size="13" lengthAdjust="spacingAndGlyphs" textLength="149" x="720" y="207.1919">SetChargingProfile.conf</text><polygon fill="#A80036" points="82,252.5234,72,256.5234,82,260.5234,78,256.5234" style="stroke: #A80036; stroke-width: 1.0;"/><line style="stroke: #A80036; stroke-width: 1.0;" x1="76" x2="692" y1="256.5234" y2="256.5234"/><text fill="#000000" font-family="sans-serif" font-size="13" lengthAdjust="spacingAndGlyphs" textLength="193" x="88" y="236.3247">POST https://client.com/12345</text><text fill="#000000" font-family="sans-serif" font-size="13" lengthAdjust="spacingAndGlyphs" textLength="244" x="88" y="251.4575">ClearProfileResult(result = ACCEPTED)</text><polygon fill="#A80036" points="686,266.5234,696,270.5234,686,274.5234,690,270.5234" style="stroke: #A80036; stroke-width: 1.0;"/><line style="stroke: #A80036; stroke-width: 1.0; stroke-dasharray: 2.0,2.0;" x1="66" x2="692" y1="270.5234" y2="270.5234"/></g></svg>
```

Comment:
Please change the PlantUML file, that is the source from which the diagram is created. This SVG file will be overwritten during the generation of the PDF.

Reply:
Done. Reverted svg changes.